src/DropDownMenu/DropDownMenu.spec.js

```diff
@@ -31,6 +32,43 @@ describe('<DropDownMenu />', () => {
     });
   });
 
+  describe('prop: disabled', () => {
+    it('disables the touchRipple effect if disableTouchRipple is undefined and disabled={true}', () => {
+      const wrapper = mountWithContext(
```

Comment:
Yes, it's really around what you are trying to test. That's fair. We could have been testing that the right property is set, but I guess that an integration test is better suited here 👍 .